### PR TITLE
Add changelog for v3.2.0

### PR DESCRIPTION
**Pull request by jsafrane**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add Changelog for v3.2.0

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.2.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+[Documentation](https://docs.k8s.io/docs/home)
+# Changelog since v3.1.0
+
+## Changes by Kind
+
+### Feature
+
+- Add configurable throughput parameters for clients to API server ([#286](https://github.com/kubernetes-csi/external-attacher/pull/286), [@RaunakShah](https://github.com/RaunakShah))
+- Bump csi translation lib version ([#293](https://github.com/kubernetes-csi/external-attacher/pull/293), [@andyzhangx](https://github.com/andyzhangx))
+- Set the value of "migrated" field in the metrics to true or false to indicate if the call is a migration enabled feature or not ([#292](https://github.com/kubernetes-csi/external-attacher/pull/292), [@nearora-msft](https://github.com/nearora-msft))
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-3.2.md:12]: I changed this in external-provisioner to word it as an ACTION REQUIRED: https://github.com/kubernetes-csi/external-provisioner/blame/master/CHANGELOG/CHANGELOG-2.2.md#L11

File: CHANGELOG/CHANGELOG-3.2.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+### Bug or Regression
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-3.2.md:14]: Is https://github.com/kubernetes-csi/external-attacher/pull/299 worth a release note?

File: CHANGELOG/CHANGELOG-3.2.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Release notes for v3.2.0
+
+[Documentation](https://docs.k8s.io/docs/home)
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply CHANGELOG/CHANGELOG-3.2.md]: Fixed

File: CHANGELOG/CHANGELOG-3.2.md (hunk 1 above)
Comment (jsafrane) [reply CHANGELOG/CHANGELOG-3.2.md]: Copied from the provisioner

**Pull request merged**